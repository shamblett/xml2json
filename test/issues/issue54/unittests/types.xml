<?xml version="1.0" encoding="utf-8"?>
<types>
  <true-boolean>true</true-boolean>
  <false-boolean>false</false-boolean>
  <object test="true" attributes="are commented in js">
    <zooi>jaja</zooi>
    <zut>frot</zut>
  </object>
  <array>
    <item/>
    <item>true</item>
    <item>2</item>
    <item>3</item>
    <item>4</item>
    <item>5</item>
    <item>six</item>
    <item>
      <key>item</key>
      <value>7</value>
    </item>
  </array>
</types>
