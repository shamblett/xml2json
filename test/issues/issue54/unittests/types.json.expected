{"types":{"true-boolean":true,"false-boolean":false,"object":{"zooi":"jaja","zut":"frot"},"array":[null,true,2,3,4,5,"six",{"key":"item","value":7}]}}
